api/src/test/java/org/pmiops/workbench/db/dao/ConceptSetDaoTest.java

```diff
@@ -0,0 +1,100 @@
+package org.pmiops.workbench.db.dao;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import java.sql.Timestamp;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.pmiops.workbench.db.model.CommonStorageEnums;
+import org.pmiops.workbench.db.model.DbConceptSet;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.db.model.DbWorkspace;
+import org.pmiops.workbench.model.Domain;
+import org.pmiops.workbench.model.Surveys;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@DataJpaTest
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+public class ConceptSetDaoTest {
+
+  @Autowired private ConceptSetDao conceptSetDao;
+  @Autowired private UserDao userDao;
+  @Autowired private WorkspaceDao workspaceDao;
+  private DbConceptSet dbConceptSet;
+
+  @Before
+  public void setUp() {
+    DbWorkspace ws = new DbWorkspace();
+    ws.setVersion(1);
+    ws = workspaceDao.save(ws);
+
+    DbUser creator = new DbUser();
+    creator.setUsername("brian");
+    creator = userDao.save(creator);
+
+    Timestamp now = new Timestamp(System.currentTimeMillis());
+    dbConceptSet =
+        conceptSetDao.save(
+            DbConceptSet.builder()
+                .addConceptSetId(1)
```

Comment:
is this an automatic ID? If so, then why specify it?

Reply:
Nice catch... must have been on autopilot.